app/src/test/kotlin/com/wire/android/ui/home/conversationslist/ConversationListViewModelTest.kt

```diff
@@ -126,8 +138,17 @@ class ConversationListViewModelTest {
         }
     }
 
+    @Test
+    fun `given a valid conversation muting state, when calling block user, then should call BlockUserUseCase`() = runTest {
+        coEvery { blockUser(any()) } returns BlockUserResult.Success
+        conversationListViewModel.blockUser(userId, "someName")
+
+        coVerify(exactly = 1) { blockUser(userId) }
+    }
```

Comment:
Wouldn't it be worth it to add a code restriction on the View model to make sure the `blockUserUseCase` doesn't get called from users that share the same team?

Reply:
it's done, "Block" option is displayed only for "non-team-members".

Reply:
also `BlockUserUseSace()` returns Error if you'll try to call it for team-mate